docs/client-development/mandate.rst

```diff
@@ -0,0 +1,526 @@
+
+    {
+        "zaak": "http://example.com",
+        "betrokkeneType": "niet_natuurlijk_persoon",
+        "roltype": "http://example.com/roltype-belanghebbende",
+        "roltoelichting": "Bedrijf dat de intermediair machtigt",
+        "indicatieMachtiging": "machtiginggever",
+        "betrokkeneIdentificatie": {
+            "kvkNummer": "99998888"
+        },
+        "authenticatieContext": null
+    }
+
+
+Query patterns
+--------------
+
+Below are examples how to request zaken, authorized by different parties.
+
+**DigiD**
+
+1. Show me my cases (based on my BSN) opened for myself:
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<own_bsn>
+             &rol__machtiging=eigen
+
+
+    Additionally ``rol.omschrijvingGeneriek`` can be used to determine the "initiator"
+    of the case.
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<own_bsn>
+             &rol__machtiging=eigen
+             &rol__omschrijvingGeneriek=initiator
+
+
+2. Show me my cases (based on my BSN) opened by an authorizee on my behalf:
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<own_bsn>
+             &rol__machtiging=machtiginggever
+
+
+    Additionally ``rol.omschrijvingGeneriek`` can be used to determine the "belanghebbende"
+    of the case.
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<own_bsn>
+             &rol__omschrijvingGeneriek=belanghebbende
+
+
+3. Show me the cases (based on my BSN) where I represent another party:
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<own_bsn>
+             &rol__machtiging=gemachtigde
+
+
+    It's also possible to filter cases based on the level of assurance and to exclude
+    results where levelOfAssurance is below some required value.
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<own_bsn>
+             &rol__machtiging=gemachtigde
+             &rol__machtiging__loa=urn:oasis:names:tc:SAML:2.0:ac:classes:MobileTwoFactorContract
+
+
+**eHerkenning**
+
+Filters for eHerkenning authorized parties depend if it's an organization or the branch, therefore
+there are two examples for each option.
+
+1. Show me my cases (based on my KVK nummer) opened for myself:
+
+    .. code::
+
+        # organization
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__nietNatuurlijkPersoon__kvk_Nummer=<own_kvk>
+             &rol__machtiging=eigen
+
+    .. code::
+
+        # branch
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__vestiging__kvkNummer=<own_kvk>
+             &rol__machtiging=eigen
```

Comment:
shouldn't this also include the branch number in the query? if a case is created by a particular branch, then employees authenticated for another branch should not see cases of the first branch, so the application needs to filter on the "current" branch

Reply:
added extra filtering `&rol__betrokkeneIdentificatie__vestiging__vestigingsNummer=<own_vestigings_nummer>`